examples/v2/chat2.nim

```diff
@@ -303,6 +304,11 @@ proc processInput(rfd: AsyncFD, rng: ref BrHmacDrbgContext) {.async.} =
 
     await node.query(HistoryQuery(contentFilters: @[HistoryContentFilter(contentTopic: DefaultContentTopic)]), storeHandler)
 
+  if conf.swap:
```

Comment:
Can you paste what you see now when booting up a `chat2` node? Both in chat app as well as in the store node.

Reply:
Probably want to enable `swap` by default too assuming it seems to work? So we can see this in logs in cluster

Reply:
`Choose a nickname >> Ud`


```
Welcome, Ud!
Connecting to nodes
Listening on
 /ip4/0.0.0.0/tcp/60003/p2p/16Uiu2HAkwCTNWUw6doqQ9FZCUJrjMPHzcASnP4MdRGnHhrWQXGGh
<May 19, 17:10> ud: Hello Ebube
<May 19, 17:10> Ebube: Hello UD
```

Reply:
The logs are currently being written to chat2.log

Reply:
Bug when I print out the balance on the terminal, I get this:

```
Choose a nickname >> Ud
Welcome, Ud!
Connecting to nodes
Listening on
 /ip4/0.0.0.0/tcp/60003/p2p/16Uiu2HAm1TmdQ6gp12ckaCVNBJxz93i8TgZ4XiDr7hZp4oeiFMoA
<May 19, 22:10> Ebube: Hello
<May 19, 22:11> Ebube: Hi
Fetching account state
Balance for 16Uiu2HAmGvuyudkxn2xv1k87J8fubZdjyzUpegX5LkJR7CmYLm5L: 2
>> Hi
<May 19, 22:13> Ud: Hi
>>
```

Should we go with having the logs in the log file alone? or should we also include them as a message in the terminal

Reply:
I get this in the storenode:

```
DBG 2021-05-19 22:13:10.899+01:00 About to accept incoming connection        topics="libp2p switch" tid=91 file=switch.nim:318
DBG 2021-05-19 22:13:10.904+01:00 created new pubsub peer                    topics="libp2p pubsub" tid=91 file=pubsub.nim:290 peer=16U*eiFMoA
DBG 2021-05-19 22:13:10.906+01:00 Incoming WakuRelay connection              topics="wakurelay" tid=91 file=waku_relay.nim:45
DBG 2021-05-19 22:13:10.908+01:00 starting pubsub read loop                  topics="libp2p pubsubpeer" tid=91 file=pubsubpeer.nim:112 conn=16U*eiFMoA:60a57f667e7ce2182ed62341 peer=16U*eiFMoA closed=false
DBG 2021-05-19 22:13:10.912+01:00 decodeMsg: decoded message                 topics="libp2p identify" tid=91 file=identify.nim:94 pubkey=secp25...73]))) addresses=@[/ip4/0.0.0.0/tcp/60003] protocols="@[\"/ipfs/id/1.0.0\", \"/vac/waku/relay/2.0.0-beta2\"]" observable_address=Some(/ip4/127.0.0.1/tcp/60001) proto_version="Some(\"ipfs/0.1.0\")" agent_version="Some(\"nim-libp2p/0.0.1\")"
DBG 2021-05-19 22:13:10.914+01:00 starting pubsub read loop                  topics="libp2p pubsubpeer" tid=91 file=pubsubpeer.nim:112 conn=16U*eiFMoA:60a57f667e7ce2182ed6233f peer=16U*eiFMoA closed=false
DBG 2021-05-19 22:13:10.917+01:00 Incoming WakuRelay connection              topics="wakurelay" tid=91 file=waku_relay.nim:45
DBG 2021-05-19 22:13:10.921+01:00 starting pubsub read loop                  topics="libp2p pubsubpeer" tid=91 file=pubsubpeer.nim:112 conn=16U*eiFMoA:60a57f667e7ce2182ed62343 peer=16U*eiFMoA closed=false
INF 2021-05-19 22:13:15.937+01:00 received query                             topics="wakustore" tid=91 file=waku_store.nim:351
INF 2021-05-19 22:13:15.939+01:00 handle store swap test                     topics="wakustore" tid=91 file=waku_store.nim:359 text=test
INF 2021-05-19 22:13:15.942+01:00 Crediting peer for                         topics="wakuswap" tid=91 file=waku_swap.nim:210 peerId=16U*eiFMoA n=2
INF 2021-05-19 22:13:15.944+01:00 Accounting state                           topics="wakuswap" tid=91 file=waku_swap.nim:215 accounting=-2
INF 2021-05-19 22:13:15.946+01:00 Disconnect threshhold not hit              topics="wakuswap" tid=91 file=waku_swap.nim:223
INF 2021-05-19 22:13:15.947+01:00 sending response                           topics="wakustore" tid=91 file=waku_store.nim:367 messages=2
DBG 2021-05-19 22:13:22.998+01:00 subscription handle                        topics="wakustore" tid=91 file=waku_store.nim:411 topic=/waku/2/default-waku/proto
```

Reply:
Log file alone is fine! I originally meant pasting the part of the log that shows it working as part of the PR description, so we can see at a glance that it is as expected. But this works too

Reply:
Usually these in-code conversations are things that can be resolved and hidden, as opposed to something that will be visiblie to people

Reply:
Yea. We can  probably resolve this conversation when done.